extensions/rich_text_components/Skillreview/directives/oppia-noninteractive-skillreview.directive.ts

```diff
@@ -55,7 +60,7 @@ angular.module('oppia').directive('oppiaNoninteractiveSkillreview', [
             // instance, then open the concept card modal. To determine if the
             // RTE is inside a CKEditor instance, check if the offsetParent
```

Comment:
This comment should go on top of `canOpenConceptCardModal` function*

Reply:
Fine.